modules/ROOT/pages/manage-as2-host-keystores.adoc

```diff
@@ -0,0 +1,63 @@
+== Managing AS2 Host Keystores
+
+AS2 host keystores consist of a public certificate and a private key. You must create an AS2 host keystore if you use AS2 endpoints.
+
+== Create an AS2 Host Keystore
+
+You can create a new AS2 keystore for the host when you configure an AS2 Receive from Partners endpoint. After you create the keystore, you can reuse it in new endpoint configurations.
+
+To create an AS2 host keystore:
+
+. In the navigation menu, select the host.
+. In the *Endpoints* section, click *New*.
+. In the *New Endpoint* window, select *Receive from partners* in the *Usage* field and *AS2* in the *Protocol* field.
+. In the *<host> keystore* section, click *Select*.
+. In the *Select a <host> keystore* window, click *New*.
+. Complete the fields in the *New keystore for <host>* window:
+.. In the *Keystore name* field, enter a unique name for the keystore.
+.. In the *<host> public certificate* field, click *Choose file*, navigate to your public certificate, and then click *Open*.
+.. In the *<host> private key file* field, click *Choose file*, navigate to your private key file, and then click *Open*.
+.. In the *Key passphrase* field, enter a passphrase for your private key.
+. Click *Upload keystore*.
++
+The new keystore appears in the *Select a <host> keystore* window and the keystore is saved. You can either continue to xref:create-endpoint.adoc[create an endpoint] or you can close the window.
+
+== Update an AS2 Host Keystore
+
+When the certificate for your AS2 keystore is about to expire, follow these steps to obtain a new one:
+
+. Depending on your organization’s policies, create a self-signed certificate and key pair, or obtain a new AS2 certificate from your certificate authority.
+. Define a date on which you plan to switch to the new certificate.
+. Share the new public certificate with your AS2 partners to give them sufficient lead time to plan the certificate switch on their side.
+
+When it's time to update the keystore with the new certificate, follow these steps:
+
+. In the navigation menu, select *<host>*.
+. In the *Certificates* section, click the keystore to update and then click *Edit*.
+. In the *<host> public certificate* field, click *Choose file*, navigate to the updated public certificate, and click *Open*.
+. In the *<host> private key file* field, click *Choose file*, navigate to the private key file, and click *Open*.
+. In the *Key passphrase* field, enter the passphrase for the new keystore.
+. Click *Update keystore*.
+. After the keystore updates, do either of the following:
+* If you have one AS2 Receive from Partners endpoint that uses the keystore, open one message flow that references the endpoint and redeploy the flow.
+* If you have more than one AS2 Receive from Partners endpoint that uses the keystore, redeploy one message flow for each endpoint that uses the keystore.
+
```

Comment:
```suggestion
```

Reply:
This AsciiDoc outdents the line after it so the line lines up with the list intro instead of the last bullet item.